### ISSUE TITLE
Reward during runtime

### ISSUE THREAD

**Issue by samiulextreem** [DLR-RM/stable-baselines3#156]
Does not show any information about running reward during runtime of any algorithm.as an example when i run this 

import gym

from stable_baselines3 import PPO

env = gym.make('CartPole-v1')

model = PPO('MlpPolicy', env, verbose=1)
model.learn(total_timesteps=10000)

obs = env.reset()
for i in range(1000):
    action, _states = model.predict(obs, deterministic=True)
    obs, reward, done, info = env.step(action)
    env.render()
    if done:
      obs = env.reset()

env.close()

the output comes as 

-----------------------------------------
| time/                   |             |
|    fps                  | 1284        |
|    iterations           | 2           |
|    time_elapsed         | 3           |
|    total_timesteps      | 4096        |
| train/                  |             |
|    approx_kl            | 0.015479716 |
|    clip_fraction        | 0.118       |
|    clip_range           | 0.2         |
|    entropy_loss         | -0.685      |
|    explained_variance   | -171        |
|    learning_rate        | 0.0003      |
|    loss                 | 6.04        |
|    n_updates            | 10          |
|    policy_gradient_loss | -0.0183     |
|    value_loss           | 48.8        |
-----------------------------------------
------------------------------------------
| time/                   |              |
|    fps                  | 1051         |
|    iterations           | 3            |
|    time_elapsed         | 5            |
|    total_timesteps      | 6144         |
| train/                  |              |
|    approx_kl            | 0.0052411165 |
|    clip_fraction        | 0.0467       |
|    clip_range           | 0.2          |
|    entropy_loss         | -0.669       |
|    explained_variance   | -55.4        |
|    learning_rate        | 0.0003       |
|    loss                 | 17.2         |
|    n_updates            | 20           |
|    policy_gradient_loss | -0.013       |
|    value_loss           | 39.1         |
------------------------------------------

i do not see any information  regarding rewards?

**Comment by Miffyli** [comment]
Please fill in the issue template, i.e. share the code you used and the setup you used. Judging by title you need to provide `verbose=1` when creating the agent.

**Comment by araffin** [comment]
>Does not show any information about running reward during runtime of any algorithm.as an example when i run this

As mentioned in the [doc](https://stable-baselines3.readthedocs.io/en/master/guide/algos.html?highlight=Monitor#rl-algorithms), you need to wrap it into a `Monitor` wrapper or simply use:
`PPO('MlpPolicy', 'CartPole-v1', verbose=1)` instead of  `PPO('MlpPolicy',  env, verbose=1)`

That way, the environment will be automatically wrapped into a `Monitor` wrapper.

Anyway, as mentioned in the doc, I recommend you to use the rl zoo for best practices: https://github.com/DLR-RM/rl-baselines3-zoo
